documentation/projects/proposals/rekognition_data/project_proposal.md

```diff
@@ -0,0 +1,142 @@
+in the Openverse catalog. This project intends augment the existing tags for the
+labeled results with the generated tags in order to improve search result
+relevancy.
+
+[aws_rekognition]: https://aws.amazon.com/rekognition/
+
+## Goals
+
+<!-- Which yearly goal does this project advance? -->
+
+Improve Search Relevancy
+
+## Requirements
+
+<!-- Detailed descriptions of the features required for the project. Include user stories if you feel they'd be helpful, but focus on describing a specification for how the feature would work with an eye towards edge cases. -->
+
+This project will be accomplished in two major pieces:
+
+1. Determining how machine-generated tags will be displayed/conveyed in the API
+   and the frontend
+2. Augmenting the catalog database with the tags we deem suitable
+
+Focusing on the frontend first may seem like putting the cart before the horse,
+but it seems prudent to imagine how the _new_ data we add will show up in both
+the frontend and the API. While both of the above will be expanded on in
+respective implementation plans, below is a short description of each piece.
+
+### Machine-generated tags in the API/Frontend
+
+The API's [`tags` field][api_tags_field] already has a spot for `accuracy`,
+along with the tag `name` itself. This is where we should include the label
+accuracy that Rekognition provides alongside the label. We should also consider
+including a new `source` section within the array of tag objects, in order to
+communicate where this accuracy value came from. In the future, we may have
+multiple instances of the same label with different `source` and `accuracy`
+values (for instance, if we chose to apply multiple machine labeling processes
+to our media records).
+
+_NB: I'm not sure if this change to the API response shape for `tags` would
+constitute an API version change. I do think having a mechanism to share tag
+source will be important going forward._
+
+[api_tags_field]:
+  https://api.openverse.engineering/v1/#tag/images/operation/images_search
+
+We should also distinguish the machine-generated tags from the creator-added
+ones in the frontend. Particularly with the introduction of the
+[additional search views](../additional_search_views/index.md), we will need to
+consider how these machine-generated tags are displayed and whether they can be
+interacted with in the same way. Similar to the API, it may also be useful to
+share the label accuracy with users (either visually or with extra content on
+mouse hover).
+
+None of the above is specific to Rekognition, but it will be necessary to
+determine for Rekognition or any other labels we wish to add in the future.
+
+### Augmenting the catalog
+
+Once we have a clear sense of how the labels will be shared downstream, we can
+incorporate the labels themselves into the catalog database. This can be broken
+down into three steps:
+
+1. Determine which labels to use
```

Comment:
+1 to Staci's request. It's worth including at the project plan level (because it involves the motivations of the project) at least a broad description of the types of tags we intend to include/exclude. We've spoken, for example, of excluding any tags related to gender or sex. Identifying the _reason_ behind that will help us make decisions across the board for any and all possible tags about whether to include them.

From a reputation safety perspective, I'd strongly encourage the folks planning this project (either at project planning or IP level) to actually go through all the generated tags and decide on some process for reviewing them. That could be on an individual-tag basis, but could also be done by some bulk method (if we used some kind of dictionary to identify categories of potentially risky words). The second method requires a clear definition of what machine generated tags we would accept and why.

Reply:
@sarayourfriend are you suggesting we might also need some mechanism for the labels themselves to be reported on a given work? I'll take some time to look over the materials we have and try to come up with a criteria/plan within this document.

Reply:
Not directly, but if someone (a provider or creator) reached out to us via some other communication channel (or, yes, used the "other" option in the content report) then we'd need to be able to take action on it.

It might not necessarily need to be implemented in the first pass at this, it could be something we state as "a future need that we need to make sure we do not accidentally make more difficult than necessary".

Reply:
As this would also apply to future _user_ generated supplemental metadata, I would wager we need a general purpose way to report inaccurate metadata. Perhaps another report type in the report form?

One other thing I think we would want to highlight here is providing the full list of tags we support (and which ones we do not include) in public documentation for the sake of transparency.

Reply:
I've added some notes about this in the document.